app/models/payments/subscription.rb

```diff
@@ -3,7 +3,7 @@ class Payments::Subscription < ApplicationRecord
 
   has_many :payments, dependent: :destroy
 
-  enum status: { canceled: 0, overdue: 1, active: 2 }
+  enum status: { canceled: 0, overdue: 1, active: 2, pending: 3 }
```

Comment:
We probably need to think if we need to account for `pending` anywhere. I've at least accounted for it in the premium determination, where pending subscriptions won't count.